src/arduboy_avr.c

```diff
@@ -133,6 +134,13 @@ void arduboy_avr_button_event(enum button_e btn_e, bool pressed)
 	}
 }
 
+void arduboy_adc_update_hook(struct avr_irq_t *irq, uint32_t value, void *param)
+{
+	avr_irq_t *iop_irq = avr_io_getirq(mod_s.avr, AVR_IOCTL_ADC_GETIRQ, ADC_IRQ_ADC1);
+	u_int16_t milivolts = (u_int16_t)(rand() % ADC_VREF_V256);
```

Comment:
Please use standard integer types i.e. `uint16_t` not `u_int16_t`

Reply:
fixed